spec/spec_helper.rb

```diff
@@ -13,10 +13,19 @@
 # it.
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+
+require 'capybara/rspec'
```

Comment:
This is adding capybara as a dependency on all of our tests, which could slow them down. We could isolate this behaviour into an `acceptance_helper.rb` that can be required only from the classes that need it.